source/x509storeissuer.c

```diff
@@ -172,7 +172,8 @@ do_x509storeissuer(size_t num)
         }
         issuer = NULL;
         counts[num]++;
-        time = ossl_time_now();
+        if ((count & 0x3f) == 0)
+            time = ossl_time_now();
```

Comment:
Please keep it aligned with other tests. If there is an issue, create a new ticket where we can discuss it.